Makefile

```diff
@@ -127,7 +133,7 @@ requirements: virtualenv $(REQUIREMENTS)
 	@echo
 	for req in $(REQUIREMENTS); do \
 		echo "Installing $$req..." ; \
-		. $(VIRTUALENV_DIR)/bin/activate && pip install -U -q -r $$req ; \
+		. $(VIRTUALENV_DIR)/bin/activate && pip install $(PIP_OPTIONS) $$req ; \
```

Comment:
I would just always put -r in front of the file since without it, it doesn't really make sense.

Anyway :P